src/Sulu/Bundle/MediaBundle/Resources/js/containers/ImageMap/hotspotsFormRenderer.scss

```diff
@@ -0,0 +1,59 @@
+@import 'sulu-admin-bundle/containers/Application/colors.scss';
+
+$hotspotsFormRendererBackgroundColor: $white;
+$hotspotsFormRendererBorderColor: $silver;
+$hotspotsFormRendererTabsBackgroundColor: transparent;
+$hotspotsFormRendererEmptyTabsLabelColor: $scorpion;
+
+.hotspots-form-renderer {
+    background-color: $hotspotsFormRendererBackgroundColor;
+    border: 1px solid $hotspotsFormRendererBorderColor;
+    border-radius: 3px;
+    width: 100%;
+    overflow: hidden;
+}
+
+.toolbar {
+    display: flex;
+}
+
+.empty-tabs-label {
+    display: flex;
+    align-items: center;
+    font-size: 12px;
+    padding-left: 10px;
+    color: $hotspotsFormRendererEmptyTabsLabelColor;
+}
+
+.tabs {
+    flex: 1 1 auto;
+    margin-bottom: -1px;
+    background-color: $hotspotsFormRendererTabsBackgroundColor !important;
```

Comment:
We were already [commenting on this](https://github.com/sulu/sulu/pull/5434#discussion_r495942333). Just got a (maybe stupid) idea in the mean time. Would maybe using `z-index` somehow help? So that stuff is shown above the border? Don't like `z-index` either, but it is definitely better than `!important` :see_no_evil:

Reply:
that doesn't work, just tried it

Reply:
I've just had another look... All the `light` theme of the `Tabs` component is doing, is to set the background to white, right? And we are only using it for the hotspots until now, is that correct? What if we would introduce a `transparent` skin instead of a `light` skin in the tabs component? Then we would not need that `!important`.

Reply:
as an alternative, we could just increase the specificity by adding another selector

Reply:
Then the problem should be fixed as well ... but I don't think `!important` is really bad in this case, since it doesn't do anything else then synthetically increasing the specificity, right?

Reply:
What speaks against making a `transparent` skin? The only thing the `light` skin is doing is setting it to white, which is then overriden anyway. If we just omit that `!important` could be removed.

And I think `!important` is bad, because it makes overriding by using a specificity impossible. I would like to leave the overriding mechanisms like specificity and `!important` to people who really want to extend our styling somehow. It should be the last escape hatch, not something that is used in our code base already, unless absolutely necessary.

And I would argue that a CSS solution without `!important` is always better than one with it.